normalizer/app/streamlit_app.py

```
@@ -140,13 +224,13 @@ def progress_callback(idx, total):
     progress_bar.progress(idx / total)
     status_text.text(f"処理中: {idx} / {total} 行")
 
-if st.button("CSV正規化を実行"):
+if st.button("CSV正規化を実行", disabled=button_disabled):
     st.session_state.log_lines = []
     st.session_state.warning_count = 0
     
     if df is not None:
         if "lat" in output_columns or "long" in output_columns:
-            if not os.environ.get("GOOGLE_MAPS_API_KEY"):
+            if mode not in ["gsi_only", "jageocoder_only"] and not os.environ.get("GOOGLE_MAPS_API_KEY"):
```

Ruff (0.12.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)